framework/shader_compiler.h

```diff
@@ -54,19 +54,21 @@ class GLSLCompiler
 	static void reset_target_environment();
 
 	/**
-	 * @brief Compiles GLSL to SPIRV code
+	 * @brief Compiles GLSL/HLSL to SPIRV code
```

Comment:
Is this correct? If so, what tool is used to compile HLSL tp SPIR-V (dxc)? Or does it simply load offline compiled SPV? If it's the latter, the comment needs to be changed.

Reply:
Glslang is used for compilation at runtime and for offline compilation DirectX Shader Compiler is being used.